plugins/cinematic/README.md

```diff
@@ -60,9 +60,9 @@ The module `KeyframeKeeper` exposes the following parameters:
 * `editSelected - animTime` (`1.0`): Edit animation time of the selected keyframe.
 * `editSelected - simTime` (`1.0`): Edit simulation time of the selected keyframe.
 * `editSelected - positionVector`: Edit the position vector of the selected keyframe.
-* `editSelected - lookatVector`: Edit the look-at vector of the selected keyframe.
-* `editSelected - resetLookat` (Assigned key: `CTRL+l`): Reset the LookAt vector of the selected keyframe.
-* `editSelected - upVector`:  Edit the up vector of the selected keyframe.
+* `editSelected - viewVector`: Edit the view vector of the selected keyframe.
```

Comment:
shouldn't we just use the notation you can see in MegaMol anyway? Like `editSelected::lookatVector` ? Or how does this appear in the GUI...